docs_app/package.json

```diff
@@ -44,94 +44,94 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.1.0",
-    "@angular/cdk": "^6.4.0",
-    "@angular/common": "^6.1.0",
-    "@angular/compiler": "^6.1.0",
-    "@angular/core": "^6.1.0",
-    "@angular/elements": "^6.1.0",
-    "@angular/forms": "^6.1.0",
-    "@angular/material": "^6.4.0",
-    "@angular/platform-browser": "^6.1.0",
-    "@angular/platform-browser-dynamic": "^6.1.0",
-    "@angular/router": "^6.1.0",
-    "@angular/service-worker": "^6.1.0",
+    "@angular/animations": "^8.0.0",
+    "@angular/cdk": "^8.0.1",
+    "@angular/common": "^8.0.0",
+    "@angular/compiler": "^8.0.0",
+    "@angular/core": "^8.0.0",
+    "@angular/elements": "^8.0.0",
+    "@angular/forms": "^8.0.0",
+    "@angular/material": "^8.0.1",
+    "@angular/platform-browser": "^8.0.0",
+    "@angular/platform-browser-dynamic": "^8.0.0",
+    "@angular/router": "^8.0.0",
+    "@angular/service-worker": "^8.0.0",
     "@stackblitz/sdk": "^1.2.0",
-    "@webcomponents/custom-elements": "^1.0.8",
+    "@webcomponents/custom-elements": "^1.2.4",
     "classlist.js": "^1.1.20150312",
-    "core-js": "^2.4.1",
-    "rxjs": "^6.2.0",
-    "tslib": "^1.9.0",
-    "web-animations-js": "^2.2.5",
-    "zone.js": "^0.8.26"
+    "core-js": "^3.1.3",
```

Comment:
Due to this upgrade, the `setup` task is broken. More specifically this breaks the `build-ie-polyfills` task since the imports in **./src/ie-polyfills.js** are not valid anymore. Change the imports to these:

```js
import 'core-js/es/symbol';
import 'core-js/es/object';
import 'core-js/es/function';
import 'core-js/es/parse-int';
import 'core-js/es/parse-float';
import 'core-js/es/number';
import 'core-js/es/math';
import 'core-js/es/string';
import 'core-js/es/date';
import 'core-js/es/array';
import 'core-js/es/regexp';
import 'core-js/es/map';
import 'core-js/es/weak-map';
import 'core-js/es/set';
```